hs/src/Reach/AST/ID/Req.hs

```diff
@@ -0,0 +1,90 @@
+{-# OPTIONS_GHC -Wno-missing-export-lists #-}
```

Comment:
I don't think this is an AST. I think this stuff goes in `hs/src/Reach/ID/{Req,Client}.hs`